nautobot/ui/package.json

```diff
@@ -14,33 +15,76 @@
         "@fortawesome/free-solid-svg-icons": "^6.2.1",
         "@fortawesome/react-fontawesome": "^0.2.0",
         "@nautobot/nautobot-ui": "github:nautobot/nautobot-ui",
+        "@pmmmwh/react-refresh-webpack-plugin": "^0.5.3",
         "@reduxjs/toolkit": "^1.9.3",
         "@rjsf/chakra-ui": "^5.6.2",
         "@rjsf/core": "^5.4.0",
         "@rjsf/utils": "^5.4.0",
         "@rjsf/validator-ajv8": "^5.4.0",
+        "@svgr/webpack": "^5.5.0",
         "@testing-library/jest-dom": "^5.16.5",
         "@testing-library/react": "^13.4.0",
         "@testing-library/user-event": "^13.5.0",
         "axios": "^1.2.1",
+        "babel-loader": "^8.2.3",
+        "babel-plugin-named-asset-import": "^0.3.8",
+        "babel-preset-react-app": "^10.0.1",
+        "bfj": "^7.0.2",
+        "browserslist": "^4.18.1",
+        "camelcase": "^6.2.1",
+        "case-sensitive-paths-webpack-plugin": "^2.4.0",
+        "css-loader": "^6.5.1",
+        "css-minimizer-webpack-plugin": "^3.2.0",
+        "dotenv": "^10.0.0",
+        "dotenv-expand": "^5.1.0",
+        "eslint": "^8.3.0",
+        "eslint-config-react-app": "^7.0.1",
+        "eslint-webpack-plugin": "^3.1.1",
+        "file-loader": "^6.2.0",
         "framer-motion": "^9.0.3",
+        "fs-extra": "^10.0.0",
+        "html-webpack-plugin": "^5.5.0",
+        "identity-obj-proxy": "^3.0.0",
+        "jest": "^27.4.3",
+        "jest-resolve": "^27.4.2",
+        "jest-watch-typeahead": "^1.0.0",
+        "mini-css-extract-plugin": "^2.4.5",
+        "postcss": "^8.4.4",
+        "postcss-flexbugs-fixes": "^5.0.2",
+        "postcss-loader": "^6.2.1",
+        "postcss-normalize": "^10.0.1",
+        "postcss-preset-env": "^7.0.1",
+        "prompts": "^2.4.2",
         "react": "^18.2.0",
+        "react-app-polyfill": "^3.0.0",
+        "react-dev-utils": "^12.0.1",
         "react-dom": "^18.2.0",
         "react-icons": "^4.7.1",
         "react-redux": "^8.0.5",
+        "react-refresh": "^0.11.0",
         "react-router-dom": "^6.4.3",
-        "react-scripts": "5.0.1",
         "redux-persist": "^6.0.0",
+        "resolve": "^1.20.0",
+        "resolve-url-loader": "^4.0.0",
+        "sass-loader": "^12.3.0",
+        "semver": "^7.3.5",
+        "source-map-loader": "^3.0.0",
+        "style-loader": "^3.3.1",
         "swr": "^1.3.0",
-        "web-vitals": "^2.1.4"
+        "tailwindcss": "^3.0.2",
+        "terser-webpack-plugin": "^5.2.5",
+        "web-vitals": "^2.1.4",
+        "webpack": "^5.64.4",
+        "webpack-dev-server": "^4.6.0",
```

Comment:
We don't use tailwind and it seems that CRA has dev dependencies as direct dependencies (probably because if it didn't they would never get installed, as they would show up a dev dependences of react-scripts/cra).

Can we quickly review things and move the obvious ones down to dev dependencies?